src/chrome/content/rules/Mrlocksmith.com.xml

```diff
@@ -0,0 +1,14 @@
+<!--
+	520 Unknown Error:
+		- mrlocksmith.com, equivalent to www.mrlocksmith.com
```

Comment:
Can you check if that's still the case. I can't reproduce.

Reply:
Neither can I. Fixing.
Likely was a problem with their origin server behind Cloudflare.